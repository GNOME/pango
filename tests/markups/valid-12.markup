<span foreground="blue" size="200%">Blue text</span> is <span style="italic">cool</span>!
