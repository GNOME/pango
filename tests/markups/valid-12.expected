Blue text is cool!


---

range 0 9
[0 9] scale 2.000000
[0 9] foreground #00000000ffff
range 9 13
range 13 17
[13 17] style 2
range 17 2147483647


---

[0:9] (null) Normal 0
[9:13] (null) Normal 0
[13:17] (null) Italic 0
[17:2147483647] (null) Italic 0
